coral/src/app/pages/topics/index.test.tsx

```diff
@@ -60,6 +64,47 @@ describe("Topics", () => {
       expect(headline).toBeVisible();
     });
 
+    it("renders 'Request A New Topic' primary action", async () => {
+      const primaryAction = screen.getByRole("button", {
+        name: "Request A New Topic",
+      });
+
+      expect(primaryAction).toBeVisible();
+    });
+
+    it("renders 'Request A New Topic' primary action", async () => {
```

Comment:
Pretty unsure about my approach for this test, would be grateful for feedback ^^